Source/buildbindinggo.go

```diff
@@ -396,7 +400,7 @@ func buildGoClass(component ComponentDefinition, class ComponentDefinitionClass,
 	w.Writeln("type %s struct {", class.ClassName)
 	if component.Global.BaseClassName == class.ClassName {
 		w.Writeln("  _     [0]func() // uncomparable; to make == not compile")
-		w.Writeln("  ref   ref       // identifies a C value, see ref type")		
+		w.Writeln("  Ref   ref       // identifies a C value, see ref type")		
```

Comment:
@alexanderoster this is an implementation detail and the downstream user should not mess with it, why making it public?

Reply:
Hi @qmuntal, thank you for the hint. I would love to have a better solution to it, but I fear I needed to make the Refs public in order to allow component injection to work. But this is still an open issue. I have merged the different code streams in develop for now in order to keep the branch confusion a bit to a minimum, and I would open the following issues:
- CGo now is using dynamic loading - find a concept to dynamically/statically link the code
- Callbacks are not implemented properly.
- Pointer variables are not implemented properly.
- Component injection works only with slight custom editing of the headers and passing raw Refs around, but this is not a good long-term solution.

Thank you for all your support!